template-manager-cmtemplate/src/main/resources/definitions/cm-cloud-storage-location-specification.json

```diff
@@ -4,7 +4,7 @@
       "propertyName": "zeppelin.notebook.s3.bucket",
       "propertyFile": "zeppelin-site",
       "description": "The directory where Zeppelin notebooks are saved",
-      "defaultPath": "{{{ defaultPath }}}/zeppelin/notebook",
+      "defaultPath": "{{{ defaultPath }}}/{{{ clusterName }}}/zeppelin",
       "propertyDisplayName": "Zeppelin Notebooks Root Directory",
```

Comment:
The change here is added `clusterName` to path and removed "`notebook`" since that is added automatically by zeppelin for s3a. Also from this path s3 bucket is extracted and used as `"zeppelin.notebook.s3.bucket"` and remaining object paths are initialized as `"zeppelin.notebook.s3.user"`. So effectively notebooks are stored in `"defaultPath/clusterName/zeppelin/notebook"`